.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
As an exploration/enchancement matter, can you think of any other handful options you can pass to `.rspec`?